### PR TITLE
[IMP] mrp_bom_through_attributes: expected singleton error when does not exist component for a planned product

### PR DESCRIPTION
**Pull request by agaldona**

### CODE REVIEW

**Comment by agaldona** [comment]
@odoomrp/developers  review please

File: mrp_bom_through_attributes/wizard/change_production_qty.py [hunk 1]
```diff
@@ -18,5 +18,6 @@ def change_prod_qty(self):
         for product_line in production.product_lines:
             move = production.move_lines.filtered(
                 lambda x: x.product_id.id == product_line.product_id.id)
-            move.product_uom_qty = product_line.product_qty
+            if move:
+                move.write({'product_uom_qty': product_line.product_qty})
```
Comment (pedrobaeza) [inline mrp_bom_through_attributes/wizard/change_production_qty.py]: Si pones write no necesitas el `if move:`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: mrp_bom_through_attributes/wizard/change_production_qty.py (hunk 1 above)
Comment (oihane) [reply mrp_bom_through_attributes/wizard/change_production_qty.py]: incluso si no hay move?? y a la hora de asignar casca??

Comment (agaldona) [reply mrp_bom_through_attributes/wizard/change_production_qty.py]: Lo he comprobado si, y no salta el error. He subido el cambio.

**Review by pedrobaeza — APPROVED** [review]

**Comment by Daniel-CA** [comment]
:+1:

**Pull request merged**